ocaml/xapi-blobstore/test/dune

```
@@ -0,0 +1,24 @@
+(executable
+ (name xapi_blobstore_test)
```

The test should be appended to a package, so it runs only for one package, and dependencies can be defined in the opam file
```suggestion
 (name xapi_blobstore_test)
 (package xapi)
```